react-app/src/hooks/api/useBuildingsApi.ts

```diff
@@ -3,12 +3,13 @@ import { IFetch } from '../useFetch';
 import { BaseEntityInterface } from '@/interfaces/IBaseEntity';
 import { EvaluationKey } from '@/interfaces/IEvaluationKey';
 import { FiscalKey } from '@/interfaces/IFiscalKey';
+import { DeepPartial } from 'react-hook-form';
 
 export interface BuildingEvaluation extends BaseEntityInterface {
   BuildingId: number;
   Building?: Building;
   Year: number;
-  Value: string;
+  Value: string | number;
```

Comment:
Just to confirm, it's being received as a string but sent as a number? We should standardize this.